### DISCUSSION TITLE
Animated gif performance

### DISCUSSION THREAD

**Discussion by jjhuff** [imgproxy/imgproxy#1073] [General]
Our users have a significant number of animated gifs. Some of them are fairly large with lots of frames. imgproxy kinda chokes on these (25+sec for simple resize operations).

My guess that that this is due to how libvips handles these cases (extracting the whole thing into a filmstrip).

Thoughts?

**Comment by DarthSim** [comment]
Hey @jjhuff!
This is not due to how libvips handles GIFs, that's because they are, as you said, fairly large with lots of frames. If you have a 1920x1080 GIF with 100 frames, this means that imgproxy basically needs to process 100 Full HD images.
